src/components/TopBar/CallButton.vue

```diff
@@ -254,6 +254,7 @@ export default {
 
 		showStartCallButton() {
 			return this.callEnabled
+				&& this.conversation.type !== CONVERSATION.TYPE.NOTE_TO_SELF
```

Comment:
`canStartCall: true`
Evereything is good here, in normal cases, if false, button is disabled, for this conv.type it should be hidden

Maybe also should be fixed on backend? cc @nickvergessen

Reply:
Fixed `canStartCall` to be false in https://github.com/nextcloud/spreed/pull/10606
But we should not only disable the button, but still hide it, so this here is still desired